### DISCUSSION TITLE
UNRAID 的挂载

### DISCUSSION THREAD

**Discussion by BattleTagY** [AlistGo/alist#246] [Q&A] [answered]
Unraid用docker安装后怎么挂载文件夹啊，路径一直错

**Comment by xhofe** [comment, marked as answer]
请知悉：docker容器内无法直接访问宿主机文件
如需挂载宿主机文件夹，在创建容器时使用`-v 宿主机路径:容器内路径`进行目录映射
具体请自行搜索：`docker 目录映射`

**Reply by BattleTagY** [reply]
请问Alist 仅需要挂载根目录即可？不需要config与temp是么

**Reply by xhofe** [reply]
没懂你问的什么什么意思。

**Reply by BattleTagY** [reply]
问题已解决，感谢您花费时间解决我的问题.十分感谢！

**Reply by xhofe** [reply]
不客气。
